specification/advisor/resource-manager/Microsoft.Advisor/2017-04-19/swagger/advisor.json

```diff
@@ -6,18 +6,277 @@
 		"description": "REST APIs for Azure Advisor"
 	},
```

Comment:
No, this spacing is not ok.

If you are generating this, you have to change the output or put it thru a filter and get the indentation down to 2 spaces.

Reply:
Fixed with latest change.